lib/net.js

```diff
@@ -1326,84 +1326,71 @@ function listen(self, address, port, addressType, backlog, fd, exclusive) {
 
 
 Server.prototype.listen = function() {
-  var self = this;
+  const argsLen = arguments.length;
```

Comment:
I disagree that it has to be all or nothing for this type of change. I do not see any harm in changing it in this PR.